config/examples/Creality/Ender-3 Pro/CrealityV422/Configuration.h

```diff
@@ -113,8 +113,7 @@
  *
  * :[2400, 9600, 19200, 38400, 57600, 115200, 250000, 500000, 1000000]
  */
-#define BAUDRATE 115200
-
+#define BAUDRATE 250000
```

Comment:
All the configs have 115200 and this assumption is baked in to lots of external apps so I'd leave it at current value.

Reply:
Makes sense. I'll change it back.